src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/CalledCopyRatioSegmentCollection.java

```diff
@@ -22,7 +22,7 @@
     /**
      * CONTIG, START, END, NUM_POINTS_COPY_RATIO, MEAN_LOG2_COPY_RATIO, CALL
      */
-    enum CalledCopyRatioSegmentTableColumn {
+    public enum CalledCopyRatioSegmentTableColumn {
```

Comment:
This should also not be exposed.  See below.

Reply:
Left in by accident.  Done.

Reply:
So how do I get the column header for a call?

Reply:
Never mind.  Forcing user to specify.

Done.